index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const app = express()
+const cors = require('cors')
+const bodyParser = require('body-parser')
+app.use(cors())
+app.use(bodyParser.json({limit : '5000kb'}))
+const {ProductsRouter} = require('./routers/products-router')
+const {WishlistRouter} = require('./routers/wishlist-router')
+const {CartRouter} = require('./routers/cart-router')
+// check
+const {dbConnection} = require('./db/dbConnection')
+
+// initiate db dbConnection
+dbConnection();
+app.get('/' , (req , res) =>{
+  res.send("hello from home")
+})
+
+app.use('/products' , ProductsRouter)
+app.use('/wishlist' , WishlistRouter)
+app.use('/cart' , CartRouter)
+
```

Comment:
Syed What if the route is not found ? 
So add a middleware as `app.use((req, res) => {
    res.status(404).json({ success: false, message: "route not found on server, please check"})
  })`
if the route is not found , it will be used for that request

Reply:
thank you for this, done as said